src/array_api_stubs/_draft/elementwise_functions.py

```diff
@@ -2091,15 +2163,15 @@ def negative(x: array, /) -> array:
     """
 
 
-def nextafter(x1: array, x2: array, /) -> array:
+def nextafter(x1: Union[array, float], x2: Union[array, float], /) -> array:
```

Comment:
Ditto with copysign. int doesn't make sense in these functions because they specifically manipulate bits of floating-point numbers.

Reply:
In the text, `x1` should have a real-valued floating-point data type, and `x2` should have the same data type as `x1`.